build.py

```diff
@@ -0,0 +1,418 @@
+        ] = f"[{','.join('\"' + x + '\"' for x in self.provisioning)}]"
+        env["PKR_VAR_headless"] = "true"
+        if self.ansible_args:
+            env["PKR_VAR_ansible_extra_args"] = self.ansible_args
+        return env
+
+    def assemble_name(self):
+        """
+        Uses a naming scheme described in\n
+        https://github.com/usegalaxy-eu/vgcn/issues/78
+        """
+        name = ["vgcn"]
+        if "generic" in self.provisioning:
+            provisioning = self.provisioning
+            provisioning.remove("generic")
+            name += [f"+{'+'.join(provisioning)}"]
+        else:
+            name += [f"!generic+{'+'.join(self.provisioning)}"]
+        name += [
+            "-".join(self.template.split("-", 2)[:2]),
+            self.assemble_timestamp(),
+            subprocess.check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"])
+            .decode("ascii")
+            .strip(),
+            subprocess.check_output(["git", "rev-parse", "--short", "HEAD"])
+            .decode("ascii")
+            .strip(),
+        ]
+        if self.comment:
+            name += [self.comment]
+        return "~".join(name)
+
+    def assemble_scp_command(self):
+        return " ".join(
+            [
+                f"scp",
+                f"-i",
+                f"{self.pvt_key}",
+                f"{self.image_path}",
+                f"{SSH_USER}@{SSH_HOST}:{STATIC_DIR}/{os.path.basename(self.image_path)}",
+            ]
+        )
+
+    def assemble_ssh_command(self):
+        return " ".join(
+            [
+                f"ssh",
+                f"-i",
+                f"{self.pvt_key}",
+                f"{SSH_USER}@{SSH_HOST}",
+                f"chmod",
+                f"ugo+r",
+                f"{STATIC_DIR}{os.path.basename(self.image_path)}",
+            ]
+        )
+
+    def assemble_timestamp(self):
+        today = datetime.date.today()
+        seconds_since_midnight = time.time() - time.mktime(today.timetuple())
+        return today.strftime("%Y%m%d") + "~" + str(int(seconds_since_midnight))
+
+    def build(self):
+        self.clean_image_dir()
+        run_subprocess_with_spinner(
+            "BUILD",
+            subprocess.Popen(
+                self.assemble_packer_build_command(),
+                env=self.assemble_packer_envs(),
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                close_fds=True,
+                shell=True,
+            ),
+        )
+
+    def convert(self):
+        run_subprocess_with_spinner(
+            name="CONVERT",
+            proc=subprocess.Popen(
+                self.assemble_convert_command(),
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                close_fds=True,
+                shell=True,
+            ),
+        )
+
+    def clean_image_dir(self):
+        if os.path.exists(DIR_PATH + "/images"):
+            shutil.rmtree(DIR_PATH + "/images")
```

Comment:
```suggestion
        if (DIR_PATH / "images").exists():
            shutil.rmtree(DIR_PATH / "images")
```

(only if you converted DIR_PATH to a `pathlib.Path` object)